### PR TITLE
Revert to simpler training hyperparameters, simpler model architecture, add generation during eval

### PR DESCRIPTION
**Pull request by mparrett**
# Summary

Playing with a few things you might be interested in.

Key changes:

- Went with a smaller size for the model after running into overfitting.
- Added early stopping based on eval loss
- Reverted training hyperparams to defaults, validated with custom penalty disabled
- Incorporated custom penalty in training and generation
- Added a few more prompts to the generation along with different generation params

### CODE REVIEW

**Comment by mparrett** [comment]
CC: @rparrett

**Comment by mparrett** [comment]
@rparrett Mainly wanted to show you this commit: https://github.com/mparrett/kilter_brain_gen/pull/1/commits/af783183ce3058a226cf1e2315401ae8a178ea6a, but my fumbling with forks late at night before bed is getting the best of me.
